src/task.c

```diff
@@ -557,6 +566,44 @@ JL_DLLEXPORT jl_value_t *jl_get_current_task(void)
     return (jl_value_t*)ptls->current_task;
 }
 
+#ifdef JL_HAVE_ASYNCIFY
+JL_DLLEXPORT jl_ucontext_t *task_ctx_ptr(jl_task_t *t)
```

Comment:
Should these be exported generally? Seems odd to have a DLLExport that only applies to a very specific configuration.
Correct me if I am wrong but the `jl_schedule_task` and `jl_task_wait` exists because we don't have a good way of calling Julia functions from JS?

Reply:
Well, everything that needs to be called from the main executable needs to be DLLEXPORT'ed, that's what that flag means. In this case, the main executable happens to be javascript and that's where our js support libraries get linked. We don't really have a way to say, "make this function available", but only to this JS library. That may come with wasm/ES module integration, but not yet.